insights/parsers/lsscsi.py

```diff
@@ -0,0 +1,55 @@
+"""
+LsSCSI - command ``/usr/bin/lsscsi``
+====================================
+
+This parser reads the output of ``/usr/bin/lsscsi`` into a list of dictionarys.
+Each item is a dictionary with six keys:
+
+* ``HCTL`` - the scsi_host,channel,target_number,LUN tuple
+* ``Peripheral-Type`` - the SCSI peripheral type
+* ``Vendor`` - the vendor name
+* ``Model`` - the model name
+* ``Revision`` - the revision string
+* ``Primary-Device-Node`` - the primary device node name
+
+This dictionary is available in the ``data`` attribute.
+
+Parsing refers to http://sg.danny.cz/scsi/lsscsi.html.
+
+Sample input::
+
+    [1:0:0:0]    storage IET      Controller       0001  -
+    [1:0:0:1]    cd/dvd  QEMU     QEMU DVD-ROM     2.5+  /dev/sr0
+    [1:0:0:2]    disk    IET      VIRTUAL-DISK     0001  /dev/sdb
+    [3:0:5:0]    tape    HP       C5713A           H910  /dev/st0
+
+Examples:
+
+    >>> lsscsi = LsSCSI(context_wrap(LSSCSI_1))
+    >>> lsscsi[0]
+    {'Model': 'Controller', 'Vendor': 'IET', 'HCTL': '[1:0:0:0]', 'Peripheral-Type': 'storage', 'Primary-Device-Node': '-', 'Revision': '0001'}
+    >>> lsscsi.device_nodes
+        ['-', '/dev/sdb', '/dev/st0']
+    >>> len(lsscsi)
+    4
+    >>> lsscsi[1]['Peripheral-Type']
+    'cd/dvd'
```

Comment:
Would you mind to incorporate the example testing as used in `ls_etc.py` and `test_ls_etc.py`?

Reply:
@bfahr  I  am sorry, I did get it, could you explain this comments a little bit for me?

Reply:
If you look at that module, it includes unit testing for the *Examples* section of the docstrings and utilizes the data in `test_ls_etc.py` instead of having to use `context_wrap` in the examples.  It is also described in the documentation guidelines here: http://insights-core.readthedocs.io/en/latest/customtut_parsers.html#parser-testing

Reply:
Got it. I had changed the `docstings test`.